packages/ast-spec/package.json

```diff
@@ -77,7 +77,7 @@
           "cwd": "{projectRoot}"
         },
         "outputs": [
-          "{projectRoot}/dist/**/*.ts"
```

Comment:
This change might have been enough by itself TBH, but the explicit `inputs` ensures it